engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -420,12 +420,22 @@ public void setRequest(LogicalPlanRequest request)
   {
     private String containerId;
     private String errorMessage;
+    private String logFileName;
```

Comment:
I tried that with independent java code snippet as well as in Apex code, it doesn't automatically reads properties of inner classes.